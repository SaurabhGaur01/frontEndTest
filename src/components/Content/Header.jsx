import React from 'react';

const Header = () => (
    <header className="appHeader">
        <div className="d-flex justify-content-between">
            <h3 className="title text-white pl-3 pt-3">Find Your Car</h3>
            <div className="pt-3 pr-3"><button type="button" className="btn btn-light">Manage Bookings</button></div>
        </div>
        <div className="text-center pb-4">
            <h3 className="title text-white pl-3 pt-3">Car Hire - Search, Compare & Save</h3>
        </div>
    </header>
);

export default Header;