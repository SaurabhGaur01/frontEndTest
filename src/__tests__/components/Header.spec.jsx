import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Content/Header';

describe('Header', () => {
    it('should render correctly', () => {
        const renderedModule = shallow(<Header />);
        expect(renderedModule).toMatchSnapshot();
    });
})