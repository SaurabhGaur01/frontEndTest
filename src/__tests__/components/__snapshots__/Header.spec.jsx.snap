// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<header
  className="appHeader"
>
  <div
    className="d-flex justify-content-between"
  >
    <h3
      className="title text-white pl-3 pt-3"
    >
      Find Your Car
    </h3>
    <div
      className="pt-3 pr-3"
    >
      <button
        className="btn btn-light"
        type="button"
      >
        Manage Bookings
      </button>
    </div>
  </div>
  <div
    className="text-center pb-4"
  >
    <h3
      className="title text-white pl-3 pt-3"
    >
      Car Hire - Search, Compare & Save
    </h3>
  </div>
</header>
`;
